### PR TITLE
Add comments and update members errors

### PR DESCRIPTION
**Pull request by kyuwonk**
* Update Noah's email address
* Update Fred's info
* Add TODO comment about initializer
* Add `vendor/bundle` to gitignore

### CODE REVIEW

File: config/initializers/ensure_semester_and_settings_present.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Makes sure at least one semester and one settings model is in the system
 # before the app starts.
+# TODO: This should not run on setup
```
Comment (ethanlee16) [inline config/initializers/ensure_semester_and_settings_present.rb]: Can you maybe file this as an issue instead?

**Comment by ntalcus** [comment]
LGTM!

File: config/locales/en/members.yml [hunk 2]
```diff
@@ -337,7 +337,7 @@ en:
 
     - first_name: 'Noah'
       last_name: 'Alcus'
-      email: 'averyyip@berkeley.edu'
+      email: 'ntalcus@berkeley.edu'
```
Comment (ntalcus) [inline config/locales/en/members.yml]: That said, it would be nice if I could add my website info as well B)
ntalcus.github.io

Comment (kyuwonk) [reply config/locales/en/members.yml]: i don't think the website links are being used lmao
but i included it anyway

File: config/initializers/ensure_semester_and_settings_present.rb (hunk 1 above)
Comment (kyuwonk) [reply config/initializers/ensure_semester_and_settings_present.rb]: Added issue !

**Pull request merged**